java/sage/media/format/FormatParser.java

```diff
@@ -57,13 +57,15 @@
     { "dtsc / 0x63737464", MediaFormat.DTS },
     { "mlpa / 0x61706c6d", MediaFormat.DOLBY_HD },
     { "AC-3 / 0X332D6361", MediaFormat.AC3 },
+    { "MATROSKA,WEBM", MediaFormat.MATROSKA },
   };
 
+  private static FormatParserPlugin formatParserPluginInstance;
```

Comment:
For the lock, add another line such as:

private static Object formatParserPluginLock = new Object();